assets/prototype/application/utilities/text/changeCase/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+export const lcFirst = (str: string): string | undefined => {
+	if (is(String, str)) {
+		return str.charAt(0).toLowerCase() + str.substring(1);
+	}
+
+	return undefined;
+};
+
+export const ucFirst = (str: string): string | undefined => {
```

Comment:
Found ramda's equivalent, but that was too convoluted, see `toTitle`:
http://czyzykowski.com/posts/7-ramda-recipes.html